Sekoia.io/action_lists_cases.json

```
@@ -0,0 +1,231 @@
+{
+    "uuid": "ac749619-3763-4029-b276-d13d9df9824f",
+    "name": "Search Cases",
+    "docker_parameters": "get_cases",
+    "description": "find cases that match your filters",
+    "arguments": {
+        "$schema": "http://json-schema.org/draft-07/schema#",
+        "type": "object",
+        "title": "list_cases",
+        "properties": {
+            "match[community_uuid]": {
+                "in": "query",
+                "type": "string"
+            },
+            "match[status_uuid]": {
+                "in": "query",
+                "type": "string"
+            },
+            "match[status_name]": {
+                "in": "query",
+                "type": "string"
+            },
+            "date[created_at]": {
+                "in": "query",
+                "type": "string"
+            },
+            "match[created_by]": {
+                "in": "query",
+                "type": "string"
+            },
+            "match[tags]": {
+                "in": "query",
+                "type": "array",
+                "items": {
+                    "type": "string" 
+                }
```

According to the [documentation](https://docs.sekoia.io/developer/api/#/Investigation/Cases/get_cases_resource), this expected type is `string`:
```suggestion
                "type": "string"
```